pkg/action/cl001/ac001/crs.go

```diff
@@ -17,31 +14,30 @@ import (
 func newCRs(releases []v1alpha1.Release, host string) (v1alpha2.ClusterCRs, error) {
 	var err error
 
-	var r *release.Release
+	var p *release.Patch
 	{
-		c := release.Config{
+		c := release.PatchConfig{
 			FromEnv:     env.ReleaseVersion(),
 			FromProject: project.Version(),
 			Releases:    releases,
 		}
 
-		r, err = release.New(c)
+		p, err = release.NewPatch(c)
 		if err != nil {
 			return v1alpha2.ClusterCRs{}, microerror.Mask(err)
 		}
 	}
 
 	var crs v1alpha2.ClusterCRs
-	fmt.Println(r.Version())
 	{
 		c := v1alpha2.ClusterCRsConfig{
 			Credential:        key.Credential,
 			Domain:            key.DomainFromHost(host),
 			Description:       explainerCommand,
 			Owner:             key.Organization,
 			Region:            key.RegionFromHost(host),
-			ReleaseComponents: r.Components(),
-			ReleaseVersion:    strings.Replace(r.Version(), "v", "", -1),
```

Comment:
@njuettner I changed the approach. I would favour using the interface in a clean way if there is only this particular use case.